### PR TITLE
Vnet Peering allow virtual network access false instead of true

### PR DESCRIPTION
**Pull request by mocofound**
It was identified that when creating a vnet peering, allow_virtual_network_access = false was the result while =true was the expected result in the portal.   #1812

### CODE REVIEW

File: azurerm/resource_arm_virtual_network_peering.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func resourceArmVirtualNetworkPeering() *schema.Resource {
 			"allow_virtual_network_access": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
+				Default:  false,
```
Comment (JunyiYi) [inline azurerm/resource_arm_virtual_network_peering.go]: The default value is already `false` according to the issue.

File: azurerm/resource_arm_virtual_network_peering.go [hunk 2]
```diff
@@ -48,7 +48,7 @@ func resourceArmVirtualNetworkPeering() *schema.Resource {
 			"allow_virtual_network_access": {
 				Type:     schema.TypeBool,
 				Optional: true,
-				Computed: true,
```
Comment (JunyiYi) [inline azurerm/resource_arm_virtual_network_peering.go]: Why did you remove `Computed`?

**Comment by mocofound** [comment]
Closing;  Needs rework.
